### ISSUE TITLE
Quest that have multiple parts show only first and others as nil

### ISSUE THREAD

**Issue by Thrumbar** [mikepauer/Carbonite#335]
https://onedrive.live.com/redir?resid=50E237CE0C7C122F!15012&authkey=!AIWJx1aeZU8d_lk&v=3&ithint=photo%2cjpg
https://onedrive.live.com/redir?resid=50E237CE0C7C122F!15015&authkey=!AAIeXtdbcqFxzcE&v=3&ithint=photo%2cjpg
https://onedrive.live.com/redir?resid=50E237CE0C7C122F!15016&authkey=!ADnitYr-zUISZNQ&v=3&ithint=photo%2cjpg
https://onedrive.live.com/redir?resid=50E237CE0C7C122F!15017&authkey=!ALO0emmIab0AGiM&v=3&ithint=photo%2cjpg
https://onedrive.live.com/redir?resid=50E237CE0C7C122F!15018&authkey=!AJsXC1Xnyrv7WFk&v=3&ithint=photo%2cjpg
https://onedrive.live.com/redir?resid=50E237CE0C7C122F!15019&authkey=!AB0AdgaoNX--D8c&v=3&ithint=photo%2cjpg

The quest is a chain that has a player talk to an npc and check areas per link.  Once the first link is done you are to go to the second one.  the pat of the link you are working on appears but all other links appear as nil but should not show until the previous part is complete.

**Comment by JWMcQuire** [comment]
What version of Carbonite do you have installed? To find out, when you log in, the "General" chat (by default) should show the full version. For example, my chat shows, "Carbonite 6.2.0 B118 Loading". This is the latest version I have installed.

Also, could you please list the quests you are referencing in the screenshots? The images are a tad too dark and the writing a bit too small to see them. This will allow the devs to look at those quests in the code to see what needs to be done if anything to fix them.

**Comment by Thrumbar** [comment]
The quest in question is the Lost Lumberjacks in Shadowmoon, but there are others like it.  As to what version I have downloaded it from Curse.
Release Version 6.2.0 as stated in the Carbonite  > Main Options page
Quest Started with 
http://www.wowhead.com/quest=34820/lost-lumberjacks
The went to
http://www.wowhead.com/npc=79966/lost-packmule
Then you go to the guest that caused the nils
http://www.wowhead.com/quest=33263/gloomshade-grove  <----  This the quest that is showed on pic

Thrumbar

**Comment by JWMcQuire** [comment]
Thank you for replying, Thrumbar. The Quest that has the issue is quest #33263, "Gloomshade Grove". 

I had the first quest "Lost Lumberjacks" on one of my toons, so I was able to check it out on my version. I too had this issue and resolved it on my system. I will be making a pull request to the Dev version after posting this response. Hopefully from there it will be merged into the master for the next public release.

**Comment by JWMcQuire** [comment]
The file that included the edits I made to fix this issue has been merged into the Dev version. It also includes fixes for other quests that work the same way. I'm sure there may be other quests that had this issue, so I will be fixing those as I find them.
